backend/internal/models/team.go

```diff
@@ -2,9 +2,9 @@ package models
 
 type Team struct {
 	DbModel
-	CorrelationId string `json:"correlation_id"`
+	CorrelationId uint   `json:"correlation_id"`
```

Comment:
@Skylude Do we want the `CorrelationId` to be a string or a uint. I have told everyone uint for ID values in an Endpoint, but you know the most on this variable from listening in on the meeting with Ethan.

Reply:
String -- it will be a GUID

Reply:
I believe this should be a string. These were updated to be GUID strings.